cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -441,6 +568,22 @@ Search for properties and tags by specifying one of:
 
 - a complete or partial value, such as ``prod*``; this will return both properties and tags
 
+Since CDAP also annotates *system* metadata to entities by default as mentioned at
+:ref:`System Metadata <metadata-lineage-system-metadata>`, the following *special* search queries are also supported:
+
+- artifacts or applications containing a specific plugin: ``plugin:<plugin-name>``
+
+- programs with a specific mode: ``batch`` or ``realtime``
+
+- applications with a specific program type: ``flow:<flow-name>``, ``service:<service-name>``,
+  ``mapreduce:<mapreduce-name>``, ``spark:<spark-name>``, ``worker:<worker-name>``,
+  ``workflow:<workflow-name>``
+
+- datasets, streams or views with the specified field:
+  - field name only: ``<field-name>``
+  - field type only: ``<field-type>``
```

Comment:
yes - https://github.com/caskdata/cdap/blob/release/3.3/cdap-app-fabric/src/test/java/co/cask/cdap/metadata/MetadataHttpHandlerTest.java#L852. 

Do you see a problem with that?